pkg/cmd/kafka/topic/topiccmdutil/validators.go

```diff
@@ -1,4 +1,4 @@
-package topicutil
+package topiccmdutil
```

Comment:
We should probably start to document all pakcage names and their meaning too, this should happen at the top of the `package topiccmdutil` declaration, but to prevent duplication can be added in a doc.go file: https://go.dev/blog/godoc